lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -5,19 +5,43 @@
 var ContextDependencyHelpers = exports;
 
 ContextDependencyHelpers.create = function(Dep, range, param, expr, options) {
```

Comment:
Looks like adding a sync `require` (and probably similarly for `import`) call resets the `chunk.chunks` count to 0 (reasonable).

`MainTemplate` then doesn't bother emitting the ensure helper, but the `ContextModule` tries to call it anyway.

I guess I'll have to do the sync require test in a separate folder.